jupyterlab_gcpscheduler/src/components/action_bar.tsx

```diff
@@ -39,14 +44,52 @@ const actionBar = style({
   ...csstips.endJustified,
 });
 
+const actionBarContainer = style({
```

Comment:
Since we've created multiple styles now, you could use the `stylesheet` function and build all of the styles into a single object the way we've done for some of the other components.

Reply:
Ah alright, done.